homeassistant/components/coinbase/const.py

```diff
@@ -312,6 +314,7 @@
     "CHF": "CHF",
     "CLF": "CLF",
     "CLP": "CLP",
+    "CLV"; "CLV",
```

Comment:
```suggestion
    "CLV": "CLV",
```

Reply:
Hi @bastgau, are you able to accept the above minor correction and then this should be good to go.

Reply:
Yes I accept the modifications. I'm not able to commit the suggestion.

Reply:
I have just commited the correction. Sorry for the delay.